std_msgs/Header header
uint8[72]       au_telemetry_value
